pydy/viz/scene.py

```diff
@@ -465,59 +480,105 @@ def display(self):
     def display_ipython(self):
         """
         Method to display the visualization inside the
-        Ipython notebook. It is only supported by IPython
+        IPython notebook. It is only supported by IPython
         versions>=2.0.0
 
         """
 
+        # Raise error whenever display_ipython() is called
+        # and IPython is not installed or IPython < '2.0.0'
+        if IPython is None:
+            raise ImportError('IPython is not installed but is required. ' +
+                              'Please installed IPython >= 2.0 and try again')
+        elif ipython_less_than_3 is None:
```

Comment:
You could probably just have one variable here. If Python < 2.0 is installed or IPython isn't installed then we just make `has_ipython=False`. Is it necessary to have two variables to check this?